src/selectors/navigation.js

```diff
@@ -8,6 +8,73 @@ import * as sections from 'constants/sections'
 import { hideSelectiveService } from 'validators/selectiveservice'
 import { hideDisciplinaryProcedures } from 'validators/militarydisciplinary'
 
+const getSectionErrors = (state, props) => {
+  const { application } = state
+  const { Errors } = application
+  const { topSection, section, sectionCode } = props
+
+  if (topSection) {
+    const sectionErrors = Errors[topSection] || []
+
+    if (section.name === 'Review') return sectionErrors
+
+    if (!section.subsections) {
+      return sectionErrors.filter(
+        e => `${e.section}/${e.subsection}` === `${sectionCode}/${section.name}`
+      )
+    }
+
+    return sectionErrors.filter(e => e.subsection.includes(section.name))
+  }
```

Comment:
This code is a hacky workaround to resolve the new section config with the existing `application.Errors` data. This can be cleaned up a ton if we update the `Errors` to include new section keys/identifiers.